src/UI/Tables/Stateless.elm

```diff
@@ -187,11 +188,13 @@ desktopView renderConfig prop opt =
 headersRender :
     RenderConfig
     -> List Column
-    -> Element msg
+    -> ( String, Element msg )
 headersRender renderConfig columns =
-    Element.row
+    ( "@headers"
```

Comment:
that's a bit "dangerous" right? I mean, if we have a dynamic column title, this will prevent being redrawn right?

Reply:
Nope, this should cause the header row to be redrawn without redrawing the entire list (brothers and sisters), and only that...